docs/docs/dev_docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Quickstart
+---
+
+Get started with the Aztec Sandbox
+
+## Introduction
+
+The Aztec Sandbox is an environment for local development on the Aztec Network. It's easy to get setup with just a single, simple command, and contains all the components needed to develop and test Aztec contracts and applications.
+
+This quickstart walks you through installing the Sandbox, deploying your first Noir contract, and verifying its execution!
+
+## Installation
+
+You can run the Sandbox using either Docker or npm.
+
+**With Docker**
+To install and start the Sandbox paste the line below in a macOS Terminal or Linux shell prompt. You will need to have Docker installed on your machine.
+
+```bash
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+To install a specific version of the sandbox, you can set the environment variable `SANDBOX_VERSION``
+
+```bash
+SANDBOX_VERSION=<version> /bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+NOTE: If `SANDBOX_VERSION`` is not defined, the script will pull the latest release of the sandbox.
+
+**With npm**
+You can download and run the Sandbox package directly if you have nodejs 18 or higher installed. You will also need an Ethereum node like Anvil or Hardhat running locally on port 8545.
+
+````bash
+npx @aztec/aztec-sandbox```
+````
+
+## Deploying a contract
+
+To interact with the sandbox now that it's running locally, install the Aztec CLI:
+
+```bash
+npm install -g @aztec/cli
+```
+
+The sandbox is preloaded with two accounts. Let's assign them to shell variables. Run the following in your terminal, so we can refer to the accounts as $ALICE and $BOB from now on:
+
+```bash
+ALICE="0x183253b9bb70e447c2bddce766b199111bec335d3396cd403ff5306ddf2f8a43"
+BOB="0x112cc0f5583099bf9cf9cd3e638aff4222a688c210a07f74262d8b4b90e51f2b"
+```
+
+Start by deploying a private token contract, minting an initial supply of private tokens to Alice:
+
+```bash
+aztec-cli deploy \
+ PrivateTokenContractAbi \
+  --args 1000000 $ALICE \
+  --salt 0
+```
```

Comment:
I'd recommend using the `#include_code` macro to pull-in cli commands from a test file. We've had a few instances where the commands accidentally got broken (when they were manually written like this) - whereas tethering the commands to actual CI tests makes them more robust.
There's an example in `./cli.md` (in the same folder as this file). In fact, many of the code snippets (maybe even the whole typescript test source file) are directly relevant to what's being explained on this page.

Reply:
just added this.